src/components/RadioButton.js

```diff
@@ -7,12 +7,14 @@ const RadioButton = React.createClass({
     activeButtonStyle: React.PropTypes.object,
     buttonStyle: React.PropTypes.object,
     checked: React.PropTypes.bool,
+    color: React.PropTypes.string,
```

Comment:
We use `brandColor` internally, but since this is a public component, I would leave it as color for greater clarity.